core/templates/pages/learner-dashboard-page/new-learner-dashboard/progress-tab.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="completed-stories" *ngIf="completedStoriesList.length !== 0">
+  <div class="completed-stories-heading">
+    <p class="completed-stories-title" [innerHTML]="'I18N_LEARNER_DASHBOARD_STORIES_COMPLETED_SECTION' | translate">
+    </p>
+    <span class="completed-stories-length">
+      {{completedStoriesList.length}}
+    </span>
+    <img [src]="getStaticImageUrl('/learner_dashboard/star.svg')" alt="star image" class="oppia-svg-star-image">
+  </div>
+  <div class="scroll-content">
+    <div class="completed-stories-content" [ngStyle]="checkMobileView() && {'width': width + 'px'}">
```

Comment:
Why does the width change with completed story count? Shouldn't it be a fixed width and story summary tiles occupying rows in it?

Reply:
It's because the width should increase with the story count to support scrolling. The [mocks](https://xd.adobe.com/view/663d1b5b-82f8-4405-be22-0994333bc908-336e/screen/2081e9c7-819c-41dd-96db-f4bfa530dcd8/specs/) don't have pagination in them. We only paginate the cards on the [Community Lessons](https://xd.adobe.com/view/663d1b5b-82f8-4405-be22-0994333bc908-336e/screen/bcce4cbf-ea0c-4259-aa54-a4ed68bfcb32/specs/) page.

Reply:
Oh ok, lgtm for now then!